modules/ROOT/pages/plugins/opensource/noneditable.adoc

```diff
@@ -78,8 +74,7 @@ NOTE: Replace `mceNonEditable` with the class name you want for your non-editabl
 . From the menu bar, select `View` > `Source code`.
 . Add the non-editable class to the desired editor content. For example:
 
```

Comment:
```suggestion
+
```
This is to link the example with the list item

Reply:
Done